src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyOptions.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Ntfy;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Mickael Perraud <mikaelkael.fr@gmail.com>
+ */
+final class NtfyOptions implements MessageOptionsInterface
+{
+    private array $options;
+    public const PRIORITY_MAX = 5;
+    public const PRIORITY_URGENT = 5;
```

Comment:
Max and urgent have the same value?

Reply:
[Ntfy](https://docs.ntfy.sh/publish/#message-priority) allows 2 namings in the doc even if `NtfyOptions::PRIORITY_MAX` should be more consistent for the component. On the other hand `NtfyOptions::PRIORITY_URGENT` refers to `Notification::IMPORTANCE_URGENT`, that's why I maintained the 2 names. If I add to choice only one, I think I choose `Notification::IMPORTANCE_URGENT`

Reply:
I think urgent is 👍🏻